cms/tests/test_frontend.py

```diff
@@ -384,17 +384,15 @@ def test_copy_to_from_clipboard(self):
 
         WebDriverWait(self.driver, 10).until(lambda driver: clipboard.is_displayed())
 
-        hov = ActionChains(self.driver).move_to_element(clipboard)
-        hov.perform()
+        clipboard.find_element_by_css_selector('.cms-clipboard-numbers a:first-child').click()
 
         # necessary sleeps for making a "real" drag and drop, that works with the clipboard
-
-        time.sleep(0.1)
+        time.sleep(0.3)
```

Comment:
here it waits for modal to open
i'm pretty sure there are better ways, but my selenium fu is lacking